src/qibolab/instruments/dummy.py

```diff
@@ -38,8 +39,9 @@ def stop(self):
     def disconnect(self):
         log.info("Disconnecting dummy instrument.")
 
-    def play(self, qubits, sequence, nshots, relaxation_time, raw_adc=False):
-        time.sleep(relaxation_time * 1e-9)
+    def play(self, qubits, sequence, options):
```

Comment:
Here we may want to add some type hints.

Reply:
Something like this ?

`def play(self, qubits: dict[str, Qubit], sequence: PulseSequence, options: ExecutionResults):
`

Reply:
Yes.
For qubits it should be better to put `Dict[Union[str,int],Qubit]`

Reply:
Options shouldn't be `ExecutionParameters`?

Reply:
Yes it should be hahaha